generators/environment/templates/docker/docker-compose.yml

```diff
@@ -106,10 +106,19 @@ services:
     links:
       - db<% if (cache.external) { %><%= cache.docker.link %><% } %><% if (mail.exists) { %><%= mail.docker.link %><% } %>
     volumes:
-      - .:/var/www/
+      # Use 'rig project sync' to leverage unison-based filesystem for app code.
+      # @see <doc-url>
+      - project-sync:/var/www/
```

Comment:
I think this needs to be:
- <%= projectName %>-sync:/var/www
so we get project-specific volumes.  Same in build.yml